versioned_docs/version-0.x.0/kubeslice-cli/kubeslice-cli-overview.mdx

```diff
@@ -0,0 +1,21 @@
+# KubeSlice CLI Overview
+
+## Introduction 
+Kubeslice-cli is a command-line tool that allows you to perform Kubeslice operations on Kubernetes clusters.  It simplifies the work need to 
+install and uninstall the workloads required to run KubeSlice Controller and KubeSlice Worker in specified clusters, register those clusters
+as part of a KubeSlice multi-cluster, and administer slices across those clusters.
+
+The tool takes input in the form of .yaml files that describe the membership of clusters in the multi-cluster (the "topology" yaml file) 
+as well as the definition of slices across those clusters (the "slice" yaml file).   Given those yaml files, it simplifies installation 
+of KubeSlice by performing all of the necessary configuration steps to apply the appropriate config to each cluster. You can install KubeSlice 
+on your existing kind or cloud clusters. 
+
+Alternatively, as a shortcut to create a multicluster of kind clusters to get familiar with KubeSlice, you can use the 
+`kubeslice-cli install —profile` option.   It will create a demo topology with one controller and two worker clusters.
```

Comment:
Remove the extra space before `It will ...`

Reply:
done